src/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -154,7 +154,7 @@ function MoveItemPropertiesCtrl($sce, $q, dimStoreService, dimItemService, dimSe
   };
 
   vm.light = null;
-  vm.classType = '';
+  vm.classType = ' ';
```

Comment:
i18next didn't like the empty string, but took a space with no problem, it only affects when no classType is assigned to the object.